### PR TITLE
namespace: refactor namespace api.

### PR DESCRIPTION
**Pull request by disksing**
At the global router level, all requests towards `/pd/api/v1/classifier/*` will be redirected to current namespace classifier (if it implements `http.Handler`).

### CODE REVIEW

File: pdctl/command/store_ns_command.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const storeNsPrefix = "pd/api/v1/store_ns/%s"
+const storeNsPrefix = "pd/api/v1/classifier/store_ns/%s"
```
Comment (siddontang) [inline pdctl/command/store_ns_command.go]: maybe better to use `/classifier/table/store_ns` ?

**Comment by siddontang** [comment]
I think we should rename Namespace in pd-ctl to TableNamespace too.

**Comment by disksing** [comment]
You are right. I plan to do it after merge this PR.

**Comment by siddontang** [comment]
Got it.

LGTM

PTAL @nolouch

**Pull request merged**